lib/constants.js

```diff
@@ -27,6 +27,52 @@ exports.LOGHELP = s => console.info(getColoredMessage(`[HELP] ${s}\n`, colors.ma
 exports.LOGSUCCESS = _ => exports.LOGINFO("Success, done.");
 exports.LOGFAILURE = _ => exports.LOGERROR("Build failed.");
 
+exports.COLOR = { 
+    reset: '\x1b[0m',
+    bright: '\x1b[1m',
+    dim: '\x1b[2m',
+    underscore: '\x1b[4m',
+    blink: '\x1b[5m',
+    reverse: '\x1b[7m',
+    hidden: '\x1b[8m',
+  
+    fgBlack: '\x1b[30m',
+    fgRed: '\x1b[31m',
+    fgGreen: '\x1b[32m',
+    fgYellow: '\x1b[33m',
+    fgBlue: '\x1b[34m',
+    fgMagenta: '\x1b[35m',
+    fgCyan: '\x1b[36m',
+    fgWhite: '\x1b[37m',
+  
+    bgBlack: '\x1b[40m',
+    bgRed: '\x1b[41m',
+    bgGreen: '\x1b[42m',
+    bgYellow: '\x1b[43m',
+    bgBlue: '\x1b[44m',
+    bgMagenta: '\x1b[45m',
+    bgCyan: '\x1b[46m',
+    bgWhite: '\x1b[47m',
+};
+
+exports.logError =  (pid, msg) => {
```

Comment:
Isn't this a breaking change? The rest of the code uses CONSTANTS.LOGINFO, etc. Now those functions will be gone.

Reply:
I think it is better to add an optional param to the existing log functions called logBinary. If this param is set to true, then the incoming data is a binary buffer and should be logged as binary. Then os_cmd can use that. 

E.g. ```osProcess.stdout.on("data", data => CONSTANTS.LOGINFO(Buffer.concat(Buffer.from(`[PID:${process.pid}] `), data), true));```

Reply:
and then in LOGINFO (and other for example) 

```
exports.LOGINFO = (s, logBinary) => {
    if (!logBinary) console.info(getColoredMessage(`[INFO] ${s}\n`, colors.green));
    else process.stdout.write(_getAsBuffer(s, CONSTANTS.COLOR.fgGreen);
}

const _getAsBuffer = (s, color) => Buffer.concat(Buffer.from(`${color}[INFO] `), s, Buffer.from(`${CONSTANTS.COLOR.reset}\n`));
```
```